ansible/group_vars/ceph/site.yml

```diff
@@ -0,0 +1,166 @@
+---
```

Comment:
@Click2Cloud-Gamma hi,Gamma, Group_vars is where variables are stored, The site.yml.sample used in stable-3.0 version of ceph_ansible is the same as  this file. Why do you modify it like this?